internal/mithril-persistence/src/database/query/cardano_transaction/get_cardano_transaction.rs

```diff
@@ -132,30 +166,10 @@ mod tests {
         insert_transactions(
             &connection,
             vec![
-                CardanoTransactionRecord::new(
-                    "tx-hash-0",
-                    BlockNumber(10),
-                    SlotNumber(50),
-                    "block-hash-10",
-                ),
-                CardanoTransactionRecord::new(
-                    "tx-hash-1",
-                    BlockNumber(10),
-                    SlotNumber(51),
-                    "block-hash-10",
-                ),
-                CardanoTransactionRecord::new(
-                    "tx-hash-2",
-                    BlockNumber(11),
-                    SlotNumber(54),
-                    "block-hash-11",
-                ),
-                CardanoTransactionRecord::new(
-                    "tx-hash-3",
-                    BlockNumber(11),
-                    SlotNumber(55),
-                    "block-hash-11",
-                ),
+                transaction_record(BlockNumber(10), SlotNumber(50)),
```

Comment:
Just a remark, it feels a bit strange to see the same block number associated with different slot numbers, even though it doesn't cause any issues in the test.
This was already the case in the original test before this PR, so it's probably not an essential change.

Reply:
Yes it was already the case and it's not a problem in this test